src/pages/Community/MemberCommunityPage.tsx

```diff
@@ -103,7 +101,7 @@ export default function MemberCommunityPage({
         />
       </div>
       <div className="flex mx-auto gap-5 my-2">
-        <div className="items-center hidden md:block">
+        <div className="items-center hidden md:block w-52">
```

Comment:
Instead of making this a fixed width, let's make it a relative width i.e w-4/12 or something like that.

Reply:
relative width has a very strange scaling w-52 is like max width 52